From a374d060dc5e8e947350d2844289579a4876137c Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Sun, 19 Apr 2020 18:11:11 +0300
Subject: [PATCH] Revert "sepolicy: Relabel wifi. properties as wifi_prop"

This reverts commit d12551bf1a6e8a9ece6bbb98344a27bde7f9b3e1.

  try to fix bootloop after splash on some devices

  last_kmsg:
  init: init second stage started!
  init: init: Unable to serialize property contexts: Duplicate prefix match detected for 'wifi.'
  init: init: Failed to initialize property area
  init: SysRq : Trigger a crash
---
 prebuilts/api/29.0/private/property_contexts | 1 -
 prebuilts/api/29.0/private/system_server.te  | 3 ---
 private/property_contexts                    | 1 -
 private/system_server.te                     | 3 ---
 4 files changed, 8 deletions(-)

diff --git a/prebuilts/api/29.0/private/property_contexts b/prebuilts/api/29.0/private/property_contexts
index fce6eb6e..8456fdb3 100644
--- a/prebuilts/api/29.0/private/property_contexts
+++ b/prebuilts/api/29.0/private/property_contexts
@@ -148,7 +148,6 @@ dalvik.                 u:object_r:dalvik_prop:s0
 ro.dalvik.              u:object_r:dalvik_prop:s0
 
 # Shared between system server and wificond
-wifi.                   u:object_r:wifi_prop:s0
 wlan.                   u:object_r:wifi_prop:s0
 
 # Lowpan properties
diff --git a/prebuilts/api/29.0/private/system_server.te b/prebuilts/api/29.0/private/system_server.te
index 73f7b472..78c71f7e 100644
--- a/prebuilts/api/29.0/private/system_server.te
+++ b/prebuilts/api/29.0/private/system_server.te
@@ -635,9 +635,6 @@ set_prop(system_server, test_harness_prop)
 # Read gsid.image_running.
 get_prop(system_server, gsid_prop)
 
-# Read wifi.interface
-get_prop(system_server, wifi_prop)
-
 # Create a socket for connections from debuggerd.
 allow system_server system_ndebug_socket:sock_file create_file_perms;
 
diff --git a/private/property_contexts b/private/property_contexts
index fce6eb6e..8456fdb3 100644
--- a/private/property_contexts
+++ b/private/property_contexts
@@ -148,7 +148,6 @@ dalvik.                 u:object_r:dalvik_prop:s0
 ro.dalvik.              u:object_r:dalvik_prop:s0
 
 # Shared between system server and wificond
-wifi.                   u:object_r:wifi_prop:s0
 wlan.                   u:object_r:wifi_prop:s0
 
 # Lowpan properties
diff --git a/private/system_server.te b/private/system_server.te
index 73f7b472..78c71f7e 100644
--- a/private/system_server.te
+++ b/private/system_server.te
@@ -635,9 +635,6 @@ set_prop(system_server, test_harness_prop)
 # Read gsid.image_running.
 get_prop(system_server, gsid_prop)
 
-# Read wifi.interface
-get_prop(system_server, wifi_prop)
-
 # Create a socket for connections from debuggerd.
 allow system_server system_ndebug_socket:sock_file create_file_perms;
 
-- 
2.25.1

